contentScript.js

```diff
@@ -0,0 +1,149 @@
+(() => {
+
+	let ytLeftControls, ytPlayer;
+	let currentVid = null;  // id of the currently playing video
+	let lastVid = null;   // id of the last handled video
+	let BmsList = [];  // list of the video bookmarks
```

Comment:
follow camelcase convention like rest of naming here and just like Mark's comment, please use more complete var names